app/src/main/java/net/squanchy/service/firestore/injection/FirestoreModule.kt

```diff
@@ -10,7 +10,6 @@ import net.squanchy.service.firestore.FirestoreDbService
 class FirestoreModule {
 
     @Provides
-    @ApplicationLifecycle
```

Comment:
Any reason why we are removing this?

Reply:
It's not needed anymore, see the commit message